### PR TITLE
Add the example to deploy vLLM serve

### PR DESCRIPTION
**Pull request by AlexHe99**
This example show the step by step to deploy vLLM serve.

### CODE REVIEW

File: example/vllm-serve/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      labels:
+        app: mistral-7b
+    spec:
+      volumes:
+      - name: dev-kfd
+        hostPath:
+          path: /dev/kfd
+      - name: dev-dri
+        hostPath:
+          path: /dev/dri
```
Comment (y2kenny-amd) [inline example/vllm-serve/deployment.yaml:24]: Have you tried your example without these hostPath?  It should work without these as the device plugin does these mounting for you.

Comment (AlexHe99) [reply example/vllm-serve/deployment.yaml]: Yes. It works after remove the hostPath and mountPath of /dev/kfd and /dev/dri. Does it means these device nodes are mount to container by default by ROCm/k8s-device-pluging?

Comment (y2kenny-amd) [reply example/vllm-serve/deployment.yaml]: I believe kubelet does the mounting with information from the ROCm/k8s-device-plugin

File: example/vllm-serve/README.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        labels:
+            app: mistral-7b
+        spec:
+        volumes:
+        - name: dev-kfd
+            hostPath:
+            path: /dev/kfd
+        - name: dev-dri
+            hostPath:
+            path: /dev/dri
```
Comment (y2kenny-amd) [inline example/vllm-serve/README.md:65]: Please remove these as well.

File: example/vllm-serve/README.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+                cpu: "6"
+                memory: 6G
+                amd.com/gpu: "1"
+            volumeMounts:
+            - name: shm
+            mountPath: /dev/shm
+            - name: dev-kfd
+            mountPath: /dev/kfd
+            - name: dev-dri
+            mountPath: /dev/dri
```
Comment (y2kenny-amd) [inline example/vllm-serve/README.md:109]: Please remove

**Review by AlexHe99 — COMMENTED** [review]
README.md and Deployment.yaml has been updated by removing mount path:dev/kfd and /dev/dri. It PASS my test.

File: example/vllm-serve/README.md (hunk 3 above)
Comment (AlexHe99) [reply example/vllm-serve/README.md]: Done.

File: example/vllm-serve/README.md (hunk 2 above)
Comment (AlexHe99) [reply example/vllm-serve/README.md]: Done.

**Pull request merged**